front/lib/resources/string_ids.ts

```diff
@@ -0,0 +1,99 @@
+import type { ModelId, Result } from "@dust-tt/types";
+import { Err, Ok } from "@dust-tt/types";
+import Sqids from "sqids";
+
+import logger from "@app/logger/logger";
```

Comment:
can we move the legacy function here and rename it as legacy.

I would name this file sId.ts but not strongly held (just to limit vocabulary creep up)

Reply:
Done